web/src/lib/search/searchQuickActions.ts

```diff
@@ -0,0 +1,43 @@
+import { findMatchInBlock, findMatchInLabelledText } from "@/lib/search/helpers";
+import { Match } from "@/lib/search/typesForSearch";
+import { QuickAction } from "@/lib/types/types";
+
+export const searchQuickActions = (quickActions: QuickAction[], term: string): Match[] => {
+  const matches: Match[] = [];
+
+  for (const quickAction of quickActions) {
+    let match: Match = {
+      filename: quickAction.filename,
+      snippets: [],
+    };
+
+    const content = quickAction.contentMd.toLowerCase();
+    const name = quickAction.name.toLowerCase();
+    const cta = quickAction.callToActionText?.toLowerCase();
+    const ctaLink = quickAction.callToActionLink?.toLowerCase();
+    const filename = quickAction.filename?.toLowerCase();
+    const urlSlug = quickAction.urlSlug.toLowerCase();
+    const id = quickAction.id.toLowerCase();
+    const form = quickAction.form?.toLowerCase();
+
+    const blockTexts = [content];
+    const labelledTexts = [
+      { content: cta, label: "CTA Text" },
+      { content: ctaLink, label: "CTA Link" },
+      { content: name, label: "Name/Title" },
+      { content: filename, label: "Filename" },
+      { content: id, label: "Id" },
+      { content: urlSlug, label: "Url Slug" },
```

Comment:
[sand] In all the other search files, we're not including filename nor ID nor urlSlug as searchable values, I believe.

It's not a problem per se, but it's worth asking the content team if this is desired or not - and depending on their answer, either removing these or adding them to the rest of the searchable fields for other file types.

Also, I would argue that for user-facing language, we should write "ID" - "Id" is 
a [psychological concept](https://en.wikipedia.org/wiki/Id,_ego_and_super-ego). `Id` is used for camelCasing but not for user-facing (or content-team-facing) language, IMO.

Reply:
filename and url slug should be searchable as well.. taking out ID